index.html

```diff
@@ -214,17 +221,27 @@ <h4>Skills</h4>
             </div>
         </section>
     </section>
-    <footer>
-        <div class="footer-info">
-            <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-            <form action="https://formspree.io/f/xrgjqplp" method="POST">
-                <ul>
-                    <li><input type="text" name="name" placeholder="Full name" maxlength="30" required /></li>
-                    <li><input type="email" name="email" placeholder="Email address" required /></li>
-                    <li><textarea name="comment" id="comment" cols="30" rows="5" maxlength="500" placeholder="Enter your text here"></textarea></li>
-                    <li><button class="btn btn-style submit-form" type="submit">Get in touch</button></li>
-                </ul>
-            </form>
+    <footer id="info">
+        <div class="footer-form">
+            <div class="footer-info">
+                <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h3>
+                <form action="https://formspree.io/f/xrgjqplp" method="POST">
+                    <ul>
+                        <li><input type="text" name="name" placeholder="Full name" maxlength="30" required /></li>
+                        <li><input type="email" name="email" placeholder="Email address" required /></li>
+                    </ul>
+                    <div class="sumbt">
+                        <ol>
+                            <li>
+                                <textarea name="comment" id="comment" cols="30" rows="5" maxlength="500"
+                                    placeholder="Enter your text here"></textarea>
+                                <button class="btn btn-style submit-form" type="submit">Get in touch</button>
+                            </li>
+                        </ol>
```

Comment:
[REQUIRED] 
- [ ] Please make sure your `<inputs/>` are lined up symmetrically just like the way it is in the [chosen Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122): 

The design implemented by you:
![Screenshot from 2022-02-17 18-08-07](https://user-images.githubusercontent.com/67608876/154488936-6fac93f8-b681-48ad-b2c5-1795bd2ebc9f.png)


[Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
![Screenshot from 2022-02-17 18-08-32](https://user-images.githubusercontent.com/67608876/154488973-ea894580-6ba8-4a44-8747-c2e980b7c29a.png)

- [ ] Please kindly make sure your `mobile-version` is not getting affected by the changes your implement for the `desktop-version`, in this particular case, `mobile-version` of the web page is affected by the current changes. Please take a look at the screenshot below:

**The current look of the mobile version from the `web-view` branch:**

![Screenshot from 2022-02-17 18-15-54](https://user-images.githubusercontent.com/67608876/154489459-d0cf7a92-10c5-48ba-b822-34336573a668.png)


**The stable look of the mobile version from the `master` branch:**

![Screenshot from 2022-02-17 18-15-22](https://user-images.githubusercontent.com/67608876/154489474-a2e6c6af-aa2d-489c-b377-3e787c0a01c3.png)